tikv-client/src/main/java/com/pingcap/tikv/TiDBJDBCClient.java

```diff
@@ -67,6 +69,30 @@ public int getDelayCleanTableLock() throws IOException, SQLException {
     return (int) enableTableLock;
   }
 
+  /**
+   * get row format version from tidb
+   *
+   * @return 1 if should not encode and write with new row format.(default) 2 if encode and write
+   *     with new row format.(default on v4.0.0 cluster)
+   * @throws SQLException if a database access error occurs
+   */
+  public int getRowFormatVersion() throws SQLException {
+    List<List<Object>> result = queryTiDBViaJDBC(TIDB_ROW_FORMAT_VERSION_SQL);
+    if (result.isEmpty()) {
+      // default set to 1
+      return 1;
```

Comment:
return `TIDB_ROW_FORMAT_VERSION_DEFAULT` instead of 1 ?

Reply:
ok